cellengine/resources/population.py

```diff
@@ -1,11 +1,30 @@
 from __future__ import annotations
+from dataclasses import field
+from typing import Optional
+
+from dataclasses_json.cfg import config
+from cellengine.utils.dataclass_mixin import DataClassMixin, ReadOnly
+
+from dataclasses import dataclass
 import cellengine as ce
-from cellengine.payloads.population import _Population
 
 
-class Population(_Population):
+@dataclass
+class Population(DataClassMixin):
+
+    _id: str = field(metadata=config(field_name="_id"))
+    name: str
+    gates: str
+    unique_name: str
```

Comment:
`_id` and `unique_name` are read-only also.

Reply:
Whoop, sorry; `refactor` isn't in the SDLC, but for some reason it was in my brain as the tag for unchanged behavior. Fixed